packages/react-baseline-inputs/src/FieldSet.tsx

```diff
@@ -57,13 +58,13 @@ export const FieldSet: React.FC<FieldSetProps> = ({
 
   const errorLabelId = `${id}_error`;
 
-  return (
+  return wrapper ? (
```

Comment:
If we pass wrapper false, do we still want errors to appear? I would think so.

Reply:
This is tricky b/c without a container, there's not really a single element we can say the error "applies to", which makes `ariaDescribedBy` (and maybe `ariaLabelledBy` in this case) hard to reason about.

I also expect that one of the more common use cases for `wrapper={false}` will be unwrapping groups of buttons for layout purposes, in which case you wouldn't want individual errors for every set — you'd manually wrap all the sets with a `FieldSet` that handled the errors in a singular way.

Reply:
Okay, fair enough. At some point in the future, we should take a step back and decide if `wrapper={false}` and optional labels are something we actually want to support.